glitter/pages/models.py

```diff
@@ -43,15 +45,25 @@ class Meta(GlitterMixin.Meta):
     def get_absolute_url(self):
         return self.url
 
+    def get_latest_version(self):
+        """ Get the latest version for the page. """
+        content_type = ContentType.objects.get_for_model(self)
+        latest_version = Version.objects.filter(
```

Comment:
I think we possibly need to `.exclude(version_number=None)` for this.

If Alice publishes version 2, Bob creates a new version which he doesn't save, then this method will return Bob's unsaved version.